lib/src/state_manager/rx/rx_impl.dart

```diff
@@ -15,14 +20,26 @@ class _RxImpl<T> implements RxInterface<T> {
     return _value;
   }
 
-  bool get canUpdate {
-    return _subscriptions.length > 0;
+  /// Common to all Types [T], this operator overloading is using for
+  /// assignment, same as rx.value
+  ///
+  /// Example:
+  /// ```
+  /// var counter = 0.obs ;
+  /// counter >>= 3; // same as counter.value=3;
+  /// print(counter); // calls .toString() now
+  /// ```
+  ///
+  /// WARNING: still WIP, needs testing!
+  _RxImpl<T> operator >>(T val) {
```

Comment:
whaaaaaaaaaaaaaat I didn't even knew that such operator existed!

Okay i just googled and noticed that is a operator for bitwise stuff

this really needs testing and documentation, i'll make a issue about that

Reply:
Sure, i will add proper docs in code eventually with samples